### PR TITLE
[Cordova Test App] Add UI about memory warning

### PR DESCRIPTION
**Pull request by russelarms**
## Description

Add UI to generate OOM; 
Add UI to display the bool

## Related PRs or issues

[AB#65567](https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/65567)

### CODE REVIEW

File: cordova-plugin-generate-low-memory/src/android/LowMemory.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import org.apache.cordova.*;
+import org.json.JSONArray;
+import org.json.JSONException;
+import android.app.Activity;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Looper;
+import java.util.concurrent.atomic.AtomicInteger;
+import android.util.Log;
+public class LowMemory extends CordovaPlugin {
```
Comment (MatkovIvan) [inline cordova-plugin-generate-low-memory/src/android/LowMemory.java]:
```suggestion

public class LowMemory extends CordovaPlugin {
```

File: cordova-plugin-generate-low-memory/src/android/LowMemory.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package com.microsoft.appcenter.cordova.demo;
```
Comment (MatkovIvan) [inline cordova-plugin-generate-low-memory/src/android/LowMemory.java]: Copyright

File: cordova-plugin-generate-low-memory/src/android/LowMemoryJni.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package com.microsoft.appcenter.cordova.demo;
+
+import android.app.Activity;
+import android.widget.TextView;
+import android.os.Bundle;
+
+
```
Comment (MatkovIvan) [inline cordova-plugin-generate-low-memory/src/android/LowMemoryJni.java]:
```suggestion
```

File: cordova-plugin-generate-low-memory/src/android/LowMemoryJni.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+public class LowMemoryJni {
+
+    public static native void generateLowMemory();
+    static {
```
Comment (MatkovIvan) [inline cordova-plugin-generate-low-memory/src/android/LowMemoryJni.java]:
```suggestion

    static {
```

File: cordova-plugin-generate-low-memory/src/android/jni/lowmemory-jni.cpp [hunk 5]
```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2019 Microsoft Corporation
```
Comment (MatkovIvan) [inline cordova-plugin-generate-low-memory/src/android/jni/lowmemory-jni.cpp]: Please use template

File: cordova-plugin-generate-low-memory/www/LowMemory.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
```
Comment (guperrot) [inline cordova-plugin-generate-low-memory/www/LowMemory.js]: add copyright

**Pull request merged**